docs/toc.md

```diff
@@ -192,27 +206,12 @@
 ### [HamburgerMenu](archive/HamburgerMenu.md)
 ### [PullToRefreshListView](archive/PullToRefreshListview.md)
 ### [SlidableListItem](archive/SlidableListItem.md)
-## Helpers
-### [HttpHelper](helpers/HttpHelper.md)
-### [HttpHelperRequest](helpers/HttpHelperRequest.md)
-### [HttpHelperResponse](helpers/HttpHelperResponse.md)
```

Comment:
Don't move things out of the archive section. We have to move the files later with proper redirects which we're going to setup post 7.0 release as we didn't know how before.